wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -19,4 +19,4 @@ Once your map has been pending for a while, has a relatively high hype train, an
 
 ## Step 3: Bubbles
 
-Just because your map is bubbled doesn't mean your work is done! Don't ignore mods by normal users on your bubbled map. Although only a Beatmap Nominator or Quality Assurance Team member can rank your map, anyone can still help improve it, and a better, more thoroughly modded map means an easier job for the BNG who eventually looks at it. Once you have applied a mod on your bubble the BN who originally gave the bubble will almost always be happy to come back and reinstate it. Keep in mind that you'll also have an easier time getting your bubble noticed by BNG members if it's high priority and on the first page of pending - giving your map some kudosu stars to push it up in the list should make the wait shorter.
+Just because your map is bubbled doesn't mean your work is done! Don't ignore mods by normal users on your bubbled map. Although only a Beatmap Nominator or Quality Assurance Team member can rank your map, anyone can help improve it, and a better, more thoroughly modded map means an easier job for the BNG who eventually looks at it. Once you have applied a mod on your bubble the Beatmap Nominator who originally gave the bubble will almost always be happy to come back and reinstate it. Keep in mind that you'll also have an easier time getting your bubble noticed by BNG members if it's high priority and on the first page of pending. Having people build your hype train to push it up the list should make the wait for ranking shorter.
```

Comment:
You know BN's can still "pop" a bubble (which it's called a veto, if they don't like the map at all). These normally have drama attached along side to it and can occur huge threads as to the veto/bubble. Unless the bubble is a simple metadata problem/simple hitobject problem, then you would expect to give good evidence as to if you should fix the suggestions to the BN's veto/bubble or not.

Reply:
nomination resets aren't the same thing as vetos. the latter doesn't need to be mentioned in this article

Reply:
however, a veto is when someone Pops a map for the reason of when the BN in question is not liking that it has been bubbled. This could be stated in some little sentence linking to the glossary instead.

Reply:
Nevermind, this paragraph talks about nomination resets. Disregard